src/modules/channel_points/index.js

```diff
@@ -0,0 +1,60 @@
+const $ = require('jquery');
+const watcher = require('../../watcher');
+const settings = require('../../settings');
+
+const containerSelector = '.chat-input__buttons-container';
+const claimButtonSelector = '.claimable-bonus__icon';
+
+const bonusPointsObserver = new window.MutationObserver(mutations =>
+    mutations.forEach(mutation => {
+        for (const el of mutation.addedNodes) {
+            $(el).find(claimButtonSelector).click();
+        }
+    })
+);
+
+class ChannelPoints {
+    constructor() {
+        settings.add({
+            id: 'autoClaimBonusPoints',
+            name: 'Automatic Claim of Bonus Points',
+            defaultValue: false,
+            description: 'Automatically claim channel bonus points'
```

Comment:
let's reword this to "Auto-claim Channel Points" and "Automatically claims bonus channel points"